WBooks/Screens/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ViewController.swift
+//  WBooks
+//
+//  Created by Guido Marucci Blas on 4/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+import Core
+
+class LibraryViewController: UIViewController {
+
+    fileprivate let _view: LibraryView = LibraryView.loadFromNib()!
+    fileprivate let _viewModel: LibraryViewModel
+    
+    init(viewModel: LibraryViewModel) {
+        _viewModel = viewModel
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = _view
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        _view.tableView.delegate = self
+        _view.tableView.dataSource = self
+        _view.tableView.register(UINib(nibName: "LibraryCellView", bundle: Bundle.main), forCellReuseIdentifier: "LibraryCellView")
```

Comment:
hmmm I think core has an extension to `register` nib more easily. Check it pls

Reply:
Impolemented both Core.register and Core.dequeue instead of previous implementations.